app/views/_includes/applications/degrees.njk

```diff
@@ -1,6 +1,6 @@
 {% set degrees = application.degree %}
-<h2 class="govuk-heading-m">Degree{{ 's' if degrees.length > 1 }}</h2>
-<div class="govuk-!-margin-bottom-8">
+<h3 class="govuk-heading-m govuk-!-margin-top-5">Degree{{ 's' if degrees.length > 1 }}</h3>
```

Comment:
We could make future iterations easier if we specify a heading class rather than an amalgamation of overrides